pkgpanda/build/__init__.py

```diff
@@ -410,47 +409,6 @@ def get_docker_id(docker_name):
     return check_output(["docker", "inspect", "-f", "{{ .Id }}", docker_name]).decode('utf-8').strip()
 
 
-def hash_str(s: str):
-    hasher = hashlib.sha1()
```

Comment:
Is there a reason these functions moved to util? was it part of "it makes more sense there" or required?